### PR TITLE
Commerce 8451

### PR DESCRIPTION
**Pull request by brianikim**
@andrea-ale-sbarra ,

https://issues.liferay.com/browse/COMMERCE-8451

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java [hunk 1]
```diff
@@ -221,6 +221,16 @@ private CPSpecificationOption _updateSpecification(
 		CPSpecificationOption cpSpecificationOption =
 			_cpSpecificationOptionService.getCPSpecificationOption(id);
 
+		if (cpSpecificationOption != null) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java:224]: You should use GetterUtil

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java (hunk 1 above)
Comment (brianikim) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java]: @andrea-ale-sbarra , resolved.

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
@andrea-ale-sbarra  is this PR still needed? if not let's close it

**Pull request closed without merging**